views/locales.html

```diff
@@ -0,0 +1,22 @@
+<div class="locales" data-locales>
+  <button class="locales__toggler button button--small" data-locales-toggle aria-expanded="false">
```

Comment:
This seems like a good place to use `aria-controls` https://accessibilityresources.org/aria-controls